_posts/2020-11-03-proces-migracji-do-chmury.md

```diff
@@ -0,0 +1,233 @@
+---
+layout:    post
+title:     "Proces migracji do chmury - czyli od czego zacząć?"
+date:      2020-11-03 08:00:00 +0100
+published: true
+author:    mhoja
+image:     /assets/img/posts/2020-11-03-proces-migracji-do-chmury/clouds.jpg
+tags:
+    - cloud
+    - serverless
+    - gcp
+    - aws
+    - azure
+    - googlecloud
+---
+
+Planując migrację systemu informatycznego do chmury, zastanawiamy się *od czego zacząć*? Proces migracji składa się z kilku etapów, które pomogą nam przejść z punktu A (systemu źródłowego), do punktu B (systemu docelowego).
```

Comment:
Tutaj wrzucam uwagę ogólną dotyczącą wypozycjonowania:
nasza fraza na którą się zapinamy to "miegracja do chmury", w tym przypadku wyjąwszy tytuł i nagłówek pozwolimy sobie na jej odmianę. Czy mógłbyś kilkakrotnie w tekście, w miejscu gdy piszesz o "migracji" poszerzyć ją o "migrację do chmury"? To na pewno pozytywnie wpłynie na pozycje w wyszukiwarce.

Reply:
Sprawdziłem ile razy występuje "migracja do chmury", "migracji do chmury" i wydaje się ok, na widzę miejsca na wciśnięcie tego jeszcze gdzieś, żeby nie wyglądało to na siłę. Pozycjonowanie pozycjonowaniem, ale tekst musi być czytelny i jak dla mnie jest ok.

Reply:
Poza linkami widzę tylko raz taką frazę. Zdecydowanie zgadzam się, że ten artykuł ma radować oczy i nieść wiedzę, ale też nie sądzę, że gdyby dodać z 5,6,7x "do chmury" to by tekst stracił na czytelności, a wpłynie na jego pozycję.

Generalnie frazy te powinny zawierać około 0,8% długości tekstu (wliczywszy tytuły i nagłówki) żeby było optymalnie. Nie musisz tego dodawać, jeżeli uważasz że na tym całość straci, ale taka jest moja sugestia żeby sprawić by więcej ludzi trafiło na artykuł.

Reply:
Dodałem do pierwszych zdań w każdym etapie.